library/sha256.c

```diff
@@ -22,8 +22,31 @@
  *  http://csrc.nist.gov/publications/fips/fips180-2/fips180-2.pdf
  */
 
+#if defined(__aarch64__) && !defined(__ARM_FEATURE_CRYPTO) && \
```

Comment:
I change the position of `pragma` and add comments about why I put the workaround at the top of file.

That's to avoid possible inclusion of `arm_neon.h` in other head files.